### DISCUSSION TITLE
Can I run sbws with shadow?

### DISCUSSION THREAD

**Discussion by jcoder53** [shadow/shadow#1909] [Q&A] [answered]
Can I run sbws with shadow?

**Comment by robgjansen** [comment, marked as answer]
I am not familiar with anyone having run sbws in Shadow yet. My understanding is that you'll at least need support for the `select()` system call, which I have [an open pull request](https://github.com/shadow/shadow/pull/1910) for that should be merged by the end of the week. I'm not sure if there will be other missing functionality after select is working, but I started [a discussion](https://github.com/shadow/shadow/discussions/1899) to explain how you could track down any other missing syscalls.

**Reply by higher-order** [reply]
Is there any official instructions on how to run sbws in shadow now?
I'm also using tornettools. Is it as simple as inserting
```
- path: <path to sbws>
  args: ...
  ...
```
To the `processes` block of each 4uthority host in `shadow.config.yaml` which is generated by tornettools?
Or is it more complicated?
